visualization/app/codeCharta/util/algorithm/streetLayout/horizontalStreet.ts

```diff
@@ -174,7 +174,7 @@ export default class HorizontalStreet extends Street {
 				sum += child.width
 			} else {
 				if (child instanceof VerticalStreet) {
-					(<VerticalStreet>child).orientation = VerticalOrientation.DOWN
+					;(<VerticalStreet>child).orientation = VerticalOrientation.DOWN
```

Comment:
I guess there is no need for the ;

Reply:
I agree. In general I find a `;` at the beginning of a line irritating. It is added by our auto formatting 💁‍♂️